<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewADC2TempDialog</class>
 <widget class="QDialog" name="NewADC2TempDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>548</width>
    <height>127</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>548</width>
    <height>127</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New ADC to Temperature settings</string>
  </property>
  <widget class="QPushButton" name="Create">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnFilePath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbSelectFile">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>10</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnDescription">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>40</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>75</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>T(Celsius) =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbMul">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>71</y>
     <width>180</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>16</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>75</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* ADC +</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbAdd">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>71</y>
     <width>180</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>16</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
